src/olympia/bandwagon/models.py

```diff
@@ -30,7 +30,7 @@ class TopTags(object):
     """Descriptor to manage a collection's top tags in cache."""
 
     def key(self, obj):
-        return '%s:top-tags:%s' % (settings.CACHE_PREFIX, obj.id)
+        return 'top-tags:{id}'.format(id=obj.id)
```

Comment:
We should create an issue to remove this, I don't think the frontend cares about top tags at all...